notebooks/yolo-detection-classification/yolo-detection-classification.ipynb

```diff
@@ -0,0 +1,670 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q "openvino&gt;=2023.1.0" "ultralytics==8.3.0" opencv-python matplotlib Pillow ipywidgets --extra-index-url https://download.pytorch.org/whl/cpu
</pre><p>Update please openvino version to the latest release: 2025.3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/3080/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>